documentation/docs/getting-started/index.md

```diff
@@ -5,149 +5,121 @@ path: /getting-started/
-
-However, apps can gain more functionality via plugins. In the next section, we'll look at how to use a plugin.
+![Featured Image](getting_started_1.png)
 
 ---
 
-### Styling
-
-The basic Fusion.js boilerplate comes with [Styletron](https://github.com/rtsao/styletron) support installed. This is done via a plugin registration, which looks like this in `src/main.js`:
-
-```js
-// src/main.js
-import App from 'fusion-react';
-import Styletron from 'fusion-plugin-styletron-react';
-import React from 'react';
-
-export default () => {
-  const app = new App(<div>Hello world</div>);
+### Step 3: Say 'Hello World!'
 
-  app.register(Styletron);
-
-  return app;
-};
-```
+In the final step of this tutorial, you will rewrite the default Fusion.js demo page to make it say 'Hello World!'. To
+simplify this step, you must replace the demo code in the existing `home.js` file with new code that can render the
+'Hello World!' text in your browser. This means the routes will remain the same and no other code changes are required.
 
-Now, let's move our `<div>` element to a separate file called `src/components/root.js` and replace the div with a styled one:
+1. Open `home.js` and remove the existing code so the file is completely empty.
+2. Copy and paste the code snippet below to `home.js`. This snippet imports the React libraries used to render the
+'Hello World!' text in your browser.
 
 ```js
-// src/components/root.js
-import React from 'react';
-import {styled} from 'fusion-plugin-styletron-react';
+// src/pages/home.js
 
-const Panel = styled('div', {background: 'silver'});
+import React from ‘react’;
 
-export default <Panel>Hello</Panel>;
+const Home = () => (
 
-// src/main.js
-import App from 'fusion-react';
-import Styletron from 'fusion-plugin-styletron-react';
-
-import root from './components/root';
-
-export default () => {
-  const app = new App(root);
-
-  app.register(Styletron);
+);
 
-  return app;
-}
+export default Home;
 ```
 
-The application in your browser should automatically reload to display the silver background.
-
----
-
-### Assets
-
-Use the virtual module `assetUrl` for other asset types, such as images.
+3. Add 'Hello World!' text between `<div>` tags.
 
 ```js
-// src/components/root.js
-import React from 'react';
-import {styled} from 'fusion-plugin-styletron-react';
-import {assetUrl} from 'fusion-core';
+// src/pages/home.js
 
-const Panel = styled('div', {background: 'silver'});
+import React from ‘react’;
 
-export default (
-  <Panel>
-    <img src={assetUrl('./my-image.gif')} />
-  </Panel>
+const Home = () => (
+ <div>Hello World!</div>
 );
+
+export default Home;
 ```
 
-Note that the argument to `assetUrl` needs to be a compile-time static string literal.
+4. Finally, re-run `yarn dev` to rerun your application. As soon the server starts again, access your file by visiting
```

Comment:
Is this necessary?  Having already run `yarn dev`, I'd expect HMR to pick up these changes.  Should we call that out somewhere?

"Hot module reloading ensures your changes will be reflected in real time if your server is still running.  If you don't see them, you have probably stopped the server.  Re-run `yarn dev` to rerun your application..."

Reply:
+1

Reply:
Good catch. I'll deviate the docs here since you're right, no reason to re-run yarn dev.